tests/functional/cli/acquire/test_acquire_users.py

```diff
@@ -56,6 +56,152 @@ def fake_raw_ldap_data(nbr_users=10):
     )
 
 
+import random
+
+
+def fake_mymila_data(nbr_users=10, nbr_profs=5):
+    """
+    Return a deterministically-generated list of fake MyMila users just as
+    they would be returned by the function `load_mymila` (yet to be developped).
+    This is used for mocking the reading of a CSV file, since we don't expect
+    being able to read directly from the database itself in the short term.
+
+    Records must have some matching points with the fake LDAP data, to allow
+    for matching to be tested.
+
+    Returns a list of dictionaries, easy to convert to a dataframe.
+    """
+
+    faculty_affiliated = [
+        "Computer Science and Operations Research",
+        "Electrical and Computer Engineering",
+        "Mathematics and Statistics",
+        "Physics",
+        "Psychology",
+        "Other",
+        "",
+    ]
+    program_of_study = [
+        "Computer Science",
+        "Doctorat en Informatique",
+        "",
+    ]
+    status = [
+        "Active",
+        "Inactive",
+        "",
+    ]
+    affiliated_university = [
+        "McGill",
+        "UdeM",
+        "Samsung SAIT",
+        "",
+    ]
+
+    # by convention, first 'nbr_profs' names will be professors and the rest students
+    def mymila_entry(i: int):
+        # 2 different types of entries: prof and student
+        is_prof = i < nbr_profs
+        if is_prof:
+            membership_types = [
+                "Permanent HQP",
+                "Visiting Researcher",
+                "Collaborating Researcher",
+            ]
+            affiliation_types = [
+                "Collaborating Alumni",
+                "Collaborating researcher",
+                "HQP - Postdoctorate",
+                "visiting researcher",
+                "",
+            ]
+            supervisors = [""]
+            current_university_title = [
+                "Canada Research Chair (Tier 2) and Assistant Professor",
+                "Assistant Professor, School of Computer Science",
+                "Professeur sous octrois agrégé / Associate Research Professor",
+                "",
+            ]
+
+        else:
+            membership_types = [
+                "Research intern",
+                "",
+            ]
+            affiliation_types = [
+                "HQP - DESS",
+                "HQP - Master's Research",
+                "HQP - PhD",
+                "HQP - Professional Master's",
+                "HQP - Undergraduate",
+                "Research Intern",
+                "",
+            ]
+            supervisors = [
+                f"John Smith{i:03d}" for i in range(nbr_profs)
+            ]  # 'nbr_profs' first names for profs
+            current_university_title = [
+                "",
+            ]
+
+        first_name = "John"
+        last_name = f"Smith{i:03d}"
+        email = f"john.smith{i:03d}@mila.quebec"
+
+        return {
```

Comment:
It would be helpful to have a helper function that can generate such dict with overridable default values. 

like 

```python
def monmila_profile(**overrides):
    data = {
      'profile type' : "",
     ...
    }
    data.update(overrides)
    return data
```